app/controllers/activity_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ActivityController < ApplicationController
+  # POST /activity
+  def create
+    @activity = Activity.new({content: activity_params[:content], user: User.find(activity_params[:user])})
+
+    unless @activity.save
+      # TODO: Error Handling
```

Comment:
I think it would be ok to reload the page to show an error (because this should be an exception). That looks something like this:  
`redirect_to root_path, notice: I18n.t('confirmation.resource_creation', resource: Activity)`